python/valis/db/models.py

```diff
@@ -194,7 +194,14 @@ class CatalogModel(PeeweeBase):
 
 class CatalogResponse(CatalogModel, SDSSidFlatBase):
     """ Response model for source catalog and sdss_id information """
-    pass
+
+    parent_catalogs: dict[str, Any] = Field(..., description='The parent catalog associations for a given catalogid')
+
+    @field_serializer('parent_catalogs')
+    def serialize_parent_catalogs(v: dict[str, Any]) -> dict[str, Any]:
+        """ Serialize the parent catalogs, excluding None values."""
+
+        return {k: v for k, v in v.items() if v is not None}
```

Comment:
This will exclude none only when that setting is True during model serialization. 
```suggestion
    @field_serializer('parent_catalogs')
    def serialize_parent_catalogs(v: dict[str, Any], info: FieldSerializationInfo) -> dict[str, Any]:
        """ Serialize the parent catalogs, excluding None values."""
        if info.exclude_none:
            return {k: v for k, v in v.items() if v is not None}
        else:
            return v
```